test/transformations.jl

```diff
@@ -0,0 +1,46 @@
+@testset "Transformations" begin
+    @testset "Translation" begin
+        @testset "functor" begin
+            trans = Translation(Cartesian{2}(), [1, 2])
+            q = trans([2, 3])
+            @test coordinates(q) == [3, 5]
+            @test coordinates(q) isa Vector
+
+            trans = Translation((1, 2))
+            p = point(3, 4)
+            q = @inferred trans(p)
+            @test coordinates(p, q) == [4, 6]
+
+            trans = Translation(Spherical{2}(), (0, pi)) # becomes a rotation
```

Comment:
I find this *super* confusing! I would expect `Translation` to always be a translation in the Cartesian system.

It's true that you can get a subset of rotations with this interpretation of "translation", but you can't get all of them.